### PR TITLE
reprinted

### PR DESCRIPTION
**Pull request by beshelf**

### CODE REVIEW

File: docs/choose/UESTC/yi-wei-bi-ye-sheng.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+
+
+# 位毕业生写给成电软院新生的一封信
```
Comment (suenyiyang) [inline docs/choose/UESTC/yi-wei-bi-ye-sheng.md]: @beshelf Losing a word here. Please delete the prev blank lines BTW. 👍

**Comment by suenyiyang** [comment]
I'm going to config the repo with eslint + prettier for a regular style of markdown files. You can wait for that and try out the brand new pr template then.

**Comment by beshelf** [comment]
fine!

**Comment by suenyiyang** [comment]
@beshelf The new process is configured successfully. You can sync the latest repo, correct the document and commit again.

**Pull request merged**